### DISCUSSION TITLE
Thanks @aead for the thorough reply in issue 17377

### DISCUSSION THREAD

**Discussion by omarkohl** [minio/minio#17480] [Show and tell]
Thank you @aead for your very thorough [reply](https://github.com/minio/minio/issues/17377#issuecomment-1597519542) in #17377 :heart: . Unfortunately the issue is locked so I can't reply there. It's a relief to learn what the bug is and see it fixed in MinIO!

**Comment by rluetzner** [comment]
Good idea. 👍 I was also looking for a way to send my thanks to @aead , but couldn't think of anything.
